ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-492] Add outgoings council tax form

## Description of change
Updates Council Tax outgoings page to save payment data in `outgoings_payments`, shows council tax information on the review summary pages but only if housing type is rent/mortgage. Submission/rehydration.

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-492

Form: https://dsdmoj.atlassian.net/browse/CRIMAPP-265 (updated)
Summary: https://dsdmoj.atlassian.net/browse/CRIMAPP-495
Submission: https://dsdmoj.atlassian.net/browse/CRIMAPP-266
Rehydration: https://dsdmoj.atlassian.net/browse/CRIMAPP-267
DB Migration: https://dsdmoj.atlassian.net/browse/CRIMAPP-613

## Notes for reviewer
N/A

## Screenshots of changes (if applicable)
### Summary
<img width="559" alt="Screenshot 2024-03-07 at 03 27 43" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/47113046/1513d12c-b9d9-4de3-b24e-1413f4a99b1f">

<img width="536" alt="Screenshot 2024-03-07 at 03 26 31" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/47113046/72fe2874-1cdf-4289-bbf6-593fc5876f62">


## How to manually test the feature
Go to /steps/outgoings/does_client_pay_council_tax

Then go to /steps/outgoings/housing_payments_where_client_lives to complete housing outgoings

Then go to /steps/submission/review to see the summary

Submit an application, then as a Caseworker in Review, return the application to see the rehydration in Apply
